apps/journalfoer-soeknad/src/main/kotlin/JournalfoerSoeknadForDoffen.kt

```diff
@@ -16,13 +16,13 @@ import org.slf4j.LoggerFactory
 import java.time.Clock
 import java.time.OffsetDateTime
 
-internal class JournalfoerBarnepensjonSoeknadForDoffen(
+internal class JournalfoerSoeknadForDoffen(
     rapidsConnection: RapidsConnection,
     private val dokumentService: DokumentService,
     private val journalfoeringService: JournalfoeringService,
     private val klokke: Clock = Clock.systemUTC()
 ) : River.PacketListener {
-    private val logger = LoggerFactory.getLogger(JournalfoerBarnepensjonSoeknadForDoffen::class.java)
+    private val logger = LoggerFactory.getLogger(JournalfoerSoeknadForDoffen::class.java)
```

Comment:
kan sikkert endre klassenavn til uavhengig hvem den journalfører til?

Reply:
Skal ta det etter hvert. Siden vi enn så lenge må håndtere **gjenlevendepensjon** gjør jeg det slik for å ha litt bedre oversikt. 

Det vil også komme en større opprydning i denne appen, men vil ikke ta for mye på én gang. 

Denne PR-en skal i hovedsak bare sikre at OMS ikke får en unødvendig journalføringsoppgave i Gjenny.